Test/com/nicoles_change_maker/test/ChangeMakerTest.java

```diff
@@ -0,0 +1,235 @@
+        change = changeMaker.makeChange(.05);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForSixCents(){
+        expectedChange.put("N", 1);
+        expectedChange.put("P", 1);
+        change = changeMaker.makeChange(.06);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForSevenCents(){
+        expectedChange.put("N", 1);
+        expectedChange.put("P", 2);
+        change = changeMaker.makeChange(.07);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForEightCents(){
+        expectedChange.put("N", 1);
+        expectedChange.put("P", 3);
+        change = changeMaker.makeChange(.08);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForNineCents(){
+        expectedChange.put("N", 1);
+        expectedChange.put("P", 4);
+        change = changeMaker.makeChange(.09);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForTenCents(){
+        expectedChange.put("D", 1);
+        change = changeMaker.makeChange(.10);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForElevenCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("P", 1);
+        change = changeMaker.makeChange(.11);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForTwelveCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("P", 2);
+        change = changeMaker.makeChange(.12);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForThirteenCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("P", 3);
+        change = changeMaker.makeChange(.13);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForFourteenCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("P", 4);
+        change = changeMaker.makeChange(.14);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForFifteenCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("N", 1);
+        change = changeMaker.makeChange(.15);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForSixteenCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("N", 1);
+        expectedChange.put("P", 1);
+        change = changeMaker.makeChange(.16);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForSeventeenCents(){
+        expectedChange.put("D", 1);
+        expectedChange.put("N", 1);
+        expectedChange.put("P", 2);
+        change = changeMaker.makeChange(.17);
+        assertEquals(expectedChange, change);
+    }
+
+    @Test
+    public void testChangeForEighteenCents(){
```

Comment:
Same question as above (https://github.com/ecmendenhall/ChangeMaker/pull/1/files#r50032542) down here. 

When I started testing, I thought writing unit tests was mostly about proving correctness, but I now realize that that's not their primary purpose. In most domains we can't possibly hope to test every single input and output. Looking at these, I suspect that they are some combination of:

1) Tests that are the "red" step in a red-green-refactor cycle, which enabled you to write new production code.
2) Tests that you wrote to verify your understanding, either of Java itself or of your own code. 
3) Tests that you wrote because I told you to TDD and it seemed like you should cover everything.

Keep in mind that tests are also code, and also rot over time without attention and maintenance. Tests of type 1) are very valuable: they are likely to cover many of the edge cases in whatever problem you're trying to solve. Tests of type 2) are sometimes useful, especially if other people are likely to have the same questions you did as you wrote this code. But you should think carefully before checking in tests of type 3). Are they actually necessary, or will they just rot over time? 

The red-green-refactor cycle includes refactoring tests, too! Eventually you'll develop a good feeling for paring test suites down to a minimum number of useful cases.